Makefile.make

```diff
@@ -120,6 +120,7 @@ MLIFILES := $(sort $(GENMLIFILES) $(EXISTINGMLI))
 COQ_EXPORTED += MLSTATICFILES MLIFILES
 
 export $(COQ_EXPORTED)
+export PYTHONIOENCODING=utf-8
```

Comment:
Why not do this in `Makefile.doc` instead?

Reply:
There may be other Python code that assumes utf-8, better to fix all Python code.  I expect Python is normally installed configured with utf-8.  Recently @palmskog ran across the same encoding issue trying to get Aelectron to work in CI (IIUC).  He patched the Python code to fix it.

Reply:
OK thanks for the explanation.

Reply:
For the record, the issue with Alectryon (with patch linked) is here: https://github.com/cpitclaudel/alectryon/issues/11

There, we chose to not set the IO encoding to UTF8 globally, but rather to configure the individual operations to assume UTF8.